src/core/constants.ts

```diff
@@ -31,4 +31,7 @@ export const defaultScreenshotOptions: ScreenshotOptions = {
     isLandscape: false,
   },
   knobs: {},
+  filePattern: null,
+  imageComparison: null,
+  imageThreshold: 0.2
```

Comment:
What are `hoge` and `fuga` variables for?

Reply:
This is a part of unmade changes yet. "Usually" after the screenshooting you have to compare old and a new one(there is a lot of random text anti-alising related stuff), using [pixelmatch](https://github.com/mapbox/pixelmatch) for example. And it accepts imageThreshold.

Reply:
`storybook-chrome-screenshot` basically wants to concentrate on taking screenshots. (For wider scope of use)

There is no target for adding specific functions yet, so this time I am happy to remove this variable :pray:

Reply:
That could be a problem. We are using the same approach with puppeteer to capture screenshots, just not from a storybooks, and VERY often it generates _a bit_ different screenshots. And, imo, it'is this tool duty to detect AA releated changes in rendering and _hide_ them from git, for example. 
It is just a 4 lines of code to handle it.

Reply:
Indeed, I understood.

If we do not add the comparative function with this PR, we would like to divide it into another PR if possible :hand: